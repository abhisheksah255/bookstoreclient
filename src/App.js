import React from 'react';

const App=()=>{
    return <div>My First React Page Component</div>
}

export default App;