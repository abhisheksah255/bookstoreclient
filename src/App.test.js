import React from 'react';
import {render} from "@testing-library/react";
import App from './App';

describe("App component", ()=>{
    it("Should render app with Error Message ",()=>{
        const{asFragment}=render(<App/>);
        expect(asFragment()).toMatchSnapshot();
    });
});