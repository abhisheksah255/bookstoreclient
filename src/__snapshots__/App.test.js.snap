// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Should render app with Error Message  1`] = `
<DocumentFragment>
  <div>
    My First React Page Component
  </div>
</DocumentFragment>
`;
